js/private/test/defs_checked.bzl

```diff
@@ -1,4 +1,4 @@
-"@generated by @aspect_rules_js//js/private:translate_pnpm_lock.bzl from pnpm lock file @//example:pnpm-lock.yaml"
+"@generated by @aspect_rules_js//js/private:translate_pnpm_lock.bzl from pnpm lock file @//:pnpm-lock.yaml"
```

Comment:
it's getting annoying how long this "golden" file is - maybe we should have a separate pnpm workspace in an internal testing folder to generate something more stable. 
for example @jfirebaugh just tripped on it https://github.com/aspect-build/rules_js/pull/119#issuecomment-1139053668